packages/inscription-view/src/components/parts/common/role/RoleSelect.tsx

```diff
@@ -1,18 +1,18 @@
-import { useRoles } from '../responsible/useRoles';
+import { useRoles } from './useRoles';
 import type { BrowserValue } from '../../../browser/Browser';
 import { Flex } from '@axonivy/ui-components';
 import { useBrowser } from '../../../browser/useBrowser';
 import { usePath } from '../../../../context/usePath';
 import Browser from '../../../browser/Browser';
 import Tags from '../../../widgets/tag/Tags';
 
-type MultipleRoleSelectProps = {
+type RoleSelectProps = {
   value: string[];
   onChange: (change: string[]) => void;
   showTaskRoles?: boolean;
 };
 
-const MultipleRoleSelect = ({ value, onChange, showTaskRoles }: MultipleRoleSelectProps) => {
+const RoleSelect = ({ value, onChange, showTaskRoles }: RoleSelectProps) => {
```

Comment:
I liked MultipleRoleSelect more than just RoleSelect.
Or at least I think it should be called RolesSelect or something the like.